packages/odyssey/CHANGELOG.md

```diff
@@ -5,11 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-
-- Rebrand type and color changes
+## [0.3.0] - 2020-06-24
+
+- Rebrand color, type and spacing changes
```

Comment:
Can we break these into Added/Changed/Removed?

Reply:
Done